f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -1154,11 +1155,20 @@ def _create_single(self, resource, service, **kwargs):
                 name=urllib.quote(pool_payload['name']),
                 partition=pool_payload['partition']
             )
-            pool_resource.members_s.members.create(**payload)
-            self._shrink_payload(pool_payload,
-                                 keys_to_keep=['partition',
-                                               'name', 'loadBalancingMode'])
-            self._pool_mgr._update(bigip, pool_payload, None, None, service)
+
+            try:
```

Comment:
I think it is better to eliminate this specific kind of Exception inside `MemberManager` rather than the upper level because we only try to catch the HTTP 409 exception caused by a single member create action?

Reply:
Sorry, I did not get your meaning......